VideoLocker/src/main/java/org/edx/mobile/player/PlayerController.java

```diff
@@ -763,7 +766,9 @@ public void onClick(View v) {
 //          }
         }
     };
-
+    /**
+     * Currently unused 3/4/2015

```

Comment:
If it's just a little bit of code that isn't connected and hasn't been tested I think we should still delete it?